lively.ide/world.js

```diff
@@ -1002,9 +1002,11 @@ export class LivelyWorld extends World {
       const commentText = await $world.prompt('Enter comment');
       if (commentText) {
         let relativePosition = pt(0, 0);
-        if (evt) {
-          const xRelative = self.localize(evt.position).x / self.width;
-          const yRelative = self.localize(evt.position).y / self.height;
+        if (evt && !evt.halo) {
+          const evtPosition = pt(evt.domEvt.pageX, evt.domEvt.pageY);
```

Comment:
That is what evt.position is also based on. Is there a reason you bypass this property directly?

Reply:
Because in my case this only helds an error message:
![image](https://user-images.githubusercontent.com/49535291/119978716-42c59780-bfba-11eb-9d94-134c651f2b66.png)
When trying to directly access `evt.position` I get this error
![image](https://user-images.githubusercontent.com/49535291/119978803-5c66df00-bfba-11eb-9a35-cbb6bfdebffa.png)
and no comment is saved

Reply:
That is a bug, and should be fixed

Reply:
Should be fixed now f881a7d. Please verify if this works for you.

Reply:
yes this fixed the issue for me! thanks!